dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-invalid-routes-annotation.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: foo
+  name: qux
+  annotations:
+    zalando.org/skipper-routes: |
+      r1: Header("test") && Path("/login") -> status(200) -> "http://foo.test"
```

Comment:
why is it wrong? `Path` here is a valid predicate

Reply:
You have Path/PathSubtree also in ingress (depends of the configuration what is used for ` pathType: ImplementationSpecific`, but with some configuration this could be not allowed to have 2 tree based predicates

Reply:
but it's still a valid ingress, do we have something that will prevent user from using `Path` here?

Reply:
So there is 2 validations here, this route was going to pass ingressValidation which only parse the route, the route itself in this case is valid. Another function `addSpecRuleV1` which is called after ingressValidator would fail because what you just described (thanks for the example!).